### DISCUSSION TITLE
Canal 29 Ensenada

### DISCUSSION THREAD

**Discussion by Hannaplus4** [iptv-org/community#359] [Show and tell]
### Channel ID

Canal29Ensenada.ar

### Stream URL

https://clientes.videostreaming.ar:19360/multimediosensenada/multimediosensenada.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by Alstruit** [comment]
Possibly Geo-blocked.

**Comment by AntiPontifex** [comment]
Stream does not work: please close.

**Comment by Hannaplus4** [comment]
It's Geo-blocked  - Argentina

**Comment by AntiPontifex** [comment]
Thank you for the stream: it will be included in the next update.

**Comment by freearhey** [comment]
https://clientes.videostreaming.ar:19360/multimediosensenada/multimediosensenada.m3u8 (offline)
<img width="635" alt="image" src="https://user-images.githubusercontent.com/7253922/232236928-69060d60-4733-4b9f-9e6d-9bf7f17a633b.png">
